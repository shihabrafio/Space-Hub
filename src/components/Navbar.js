import React from 'react';
import { NavLink } from 'react-router-dom';
import links from './links';

const Navbar = () => {
  <div>
    <ul>
      {links.map((link) => (
        <li key={link.text}>
          <NavLink to={link.path}>{link.text}</NavLink>
        </li>
      ))}
    </ul>
  </div>;
};
export default Navbar;
