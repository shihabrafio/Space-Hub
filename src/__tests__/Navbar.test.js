import Navbar from '../components/Navbar';

describe('Navbar', () => {
  it('render Navbar Components', () => {
    expect(Navbar).toMatchSnapshot();
  });
});
