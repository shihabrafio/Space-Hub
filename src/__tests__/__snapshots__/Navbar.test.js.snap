// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar render Navbar Components 1`] = `[Function]`;
